helios-services/src/test/java/com/spotify/helios/agent/TaskRunnerTest.java

```diff
@@ -58,6 +69,11 @@
   @Mock private Clock clock;
   @Mock private ContainerDecorator containerDecorator;
 
+  @Mock private TaskRunner.NopListener mockListener;
```

Comment:
Yet Another Opinion on Style but I like just declaring the fields (when they need to be fields) using the expression @gimaker mentioned, not super crazy about `@Mock` as I feel like it saves you just a few keystrokes